plugins/inspect/services_inspector.rb

```diff
@@ -42,6 +47,13 @@ def has_systemd(system)
     false
   end
 
+  def has_redhat_chkconfig(system)
+    system.run_command("chkconfig", "--version")
```

Comment:
How does this check, if it has "redhat" chkconfig? It only tests for the existence of `chkconfig`, isn't it? How is this RedHat specific?